docs/controls/CameraPreview.md

```diff
@@ -25,7 +25,6 @@ The **CameraPreview** control allows to easily preview video in the MediaPlayerE
 private void CameraPreviewControl_FrameArrived(object sender, FrameEventArgs e)
 {
 	var videoFrame = e.VideoFrame;
-	var softwareBitmap = e.SoftwareBitmap;
```

Comment:
Can you revert this as this code has been modified o get software bitmap from video frame